WordPress/Classes/Extensions/NoResultsViewController+FollowedSites.swift

```diff
@@ -2,11 +2,21 @@ import Foundation
 
 extension NoResultsViewController {
     private struct Constants {
-        static let noFollowedSitesTitle = NSLocalizedString("No followed sites", comment: "Title for the no followed sites result screen")
-        static let noFollowedSitesSubtitle = NSLocalizedString("When you follow sites, you’ll see their content here.", comment: "Subtitle for the no followed sites result screen")
-        static let noFollowedSitesButtonTitle = NSLocalizedString("Discover Sites", comment: "Title for button on the no followed sites result screen")
-
-        static let noFollowedSitesImage = "wp-illustration-following-empty-results"
+        static let noFollowedSitesTitle = NSLocalizedString(
+            "reader.no.blogs.title",
+            value: "No blog subscriptions",
+            comment: "Title for the no followed blogs result screen"
+        )
+        static let noFollowedSitesSubtitle = NSLocalizedString(
+            "reader.no.blogs.subtitle",
+            value: "Subscribe to blogs in For You and you’ll see their latest posts here. Or search for a blog that you like already.",
```

Comment:
It just occurred to me that we don't have a 'For You' state. Should we use Discover for now?

```suggestion
            value: "Subscribe to blogs in Discover and you’ll see their latest posts here. Or search for a blog that you like already.",
```

Reply:
Good point, I made the change in https://github.com/wordpress-mobile/WordPress-iOS/pull/22416/commits/26f4f2f4d86df505998523a99065f1dd20068b10.